spec/support/faux_writer.rb

```diff
@@ -3,22 +3,11 @@
 require 'datadog/tracing/writer'
 
 require 'support/faux_transport'
-require 'support/network_helpers'
 
 # FauxWriter is a dummy writer that buffers spans locally.
 class FauxWriter < Datadog::Tracing::Writer
-  include NetworkHelpers
-
   def initialize(options = {})
-    if ENV['DD_AGENT_HOST'] == 'testagent' && !options[:disable_test_agent] && test_agent_running?
-      options[:transport] ||= Datadog::Transport::HTTP.default do |t|
```

Comment:
Do we ever need this piece of functionality?

Reply:
No